### PR TITLE
Updating the merged api response object to not contain confusing data.

### PR DESCRIPTION
**Pull request by THOR300**
### Correcting Merged Api Result 
--- 
Previously the merged api result still contained data like `content` and `pages` from the first api response in the array to merge. This was somewhat confusing as it is not actually merged data. 

Thus, removing and updating the tests accordingly.

### CODE REVIEW

File: src/azure_pdf_parser/utils.py [hunk 1]
```diff
@@ -104,8 +104,9 @@ def merge_responses(batches: Sequence[PDFPagesBatchExtracted]) -> AnalyzeResult:
             all_tables.extend(batch.extracted_content.tables)
         all_pages.extend(batch.extracted_content.pages)
 
-    # Copy the first result to a variable and add the content for all the pages.
-    merged_analyse_result: AnalyzeResult = batches.pop(0).extracted_content
+    merged_analyse_result = AnalyzeResult()
+    merged_analyse_result.api_version = batches[0].extracted_content.api_version
+    merged_analyse_result.model_id = batches[0].extracted_content.model_id
```
Comment (olaughter) [inline src/azure_pdf_parser/utils.py:109]: Do we use either of these? I'm wondering if it'd be smoother to just leave them out?

**Review by olaughter — APPROVED** [review]
👍  non blocking question below

File: src/azure_pdf_parser/utils.py (hunk 1 above)
Comment (joel-wright) [reply src/azure_pdf_parser/utils.py:109]: yeah, we want to be able to identify the api/model versions for later re-processing

File: tests/test_utils.py [hunk 2]
```diff
@@ -81,6 +81,10 @@ def test_merge_responses_one_page_results(
     one_page_analyse_result: AnalyzeResult,
 ) -> None:
     """Test that the responses are merged correctly."""
+    one_page_analyse_result.documents = ["test_document"]
+    one_page_analyse_result.languages = ["test_language"]
+    one_page_analyse_result.styles = ["test_style"]
```
Comment (joel-wright) [inline tests/test_utils.py:86]: I think you're gonna need to explain the rationale behind these changes in a comment just to explain what these changes allow us to verify. It's not exactly clear to me now & future-you will thank you :grin:

**Review by joel-wright — APPROVED** [review]
Looks like it will make the content of the `AnalyzeResult` we actually use a bit clearer, just the request for comments in the tests to explain.

**Pull request merged**